Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptTypesService.java

```diff
@@ -0,0 +1,46 @@
+package edu.asu.conceptpower.app.core.impl;
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.conceptpower.app.core.IConceptTypesService;
+
+/**
+ * This helper class is used for identifying the concept type based on the
+ * concept id.
+ * 
+ * @author karthikeyanmohan
+ *
+ */
+@Service
+public class ConceptTypesService implements IConceptTypesService {
+    
+    /**
+     * This method returns the concept types based on the id. If id is null then
+     * null value is returned by the method.
+     * 
+     * Concept Type is determined based on the type of id.
+     * 
+     * If id contains ??, then that concept is a generic wordnet concept.
+     * 
+     * If id starts with WID, then that concept is a specifc wordnet concept.
+     * 
+     * If both of the above condition is not met, then the concept is local CCP
+     * concept.
```

Comment:
this is true for any implementation of the interface (or at least the different types of ids). so the interface should at least list the different types.